src/pages/TxSender.vue

```diff
@@ -134,6 +155,17 @@ export default defineComponent({
 					);
 				} else encodedTx = convertBase64ToUnit8Array(this.txInput);
 			}
+			let txID = "";
+			if (!this.isJsonSelected) {
+				const decodedTxn = decodeObj(
+					convertBase64ToUnit8Array(this.txInput)
+				) as EncodedSignedTransaction;
+
+				txID = algosdk.Transaction.from_obj_for_encoding(decodedTxn.txn).txID();
+			} else {
+				// TODO : ADD TxID support for json
```

Comment:
please create a task for this in pivotal tracker

Reply:
okay